masterlist.yaml

```diff
@@ -9121,6 +9121,12 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+        name: 'Curse of the Dragonborn - REDUX'
+    group: *earlyLoadersGroup
+
```

Comment:
Now that the plugin is correctly named `OPHybrid.esp`, the complete entry should be sorted alphabetically within the `Gameplay` category. So please sort `OPHybrid.esp` inbetween `Open World Loot.esp` and `OWL Challenge Mode.esp`. After that this PR can be merged.